src/explore-education-statistics-common/src/modules/find-statistics/components/charts/__tests__/VerticalBarBlock.tsx

```diff
@@ -29,11 +29,11 @@ describe('VerticalBarBlock', () => {
     ).toBe(1);
 
     expect(
-      domUtil.elementContainingText(container, 'text tspan', '2014').length,
+      domUtil.elementContainingText(container, 'text tspan', '2014/15').length,
```

Comment:
I think any instance of this DOM util can be replaced by the more succinct `toHaveTextContext` assertion.